index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Joleen Tsai</title>
+</head>
+<link rel="stylesheet" href="assets/css/reset.css">
+<link rel="stylesheet" href="assets/css/style.css">
+
+<body>
+    <header class="header">
+        <div class="namebox">Joleen Tsai</div>
+        <div class="topbuttons">
+            <a href="index.html" class="topbuttons">About</a>
+            <a href="portfolio.html" class="topbuttons">Portfolio</a>
+            <a href="contact.html" class="topbuttons">Contact</a>
+        </div>
+    </header>
+    <div class="container">
+        <div class="indexcontent">
+            <h1 class="title">About Me</h1>
+            <img src="assets/images/Headshot.JPG" img class="headshot" alt="Headshot">
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Amet pariatur, a atque architecto explicabo iure corrupti.
+                A repellat laboriosam minima consequatur iusto? Quidem sed adipisci ab quod vel iste alias! Repudiandae hic,
+                distinctio architecto neque sequi exercitationem nemo libero. Quos aliquid blanditiis beatae maximeet facere
+                laborum cupiditate aut, excepturi voluptatibus deleniti fugiat amet, accusantium autem, asperiores odit eum
+                aperiam.
+            </p>
+            <br>
+            <p>
+                Consectetur facere a, quos aperiam quia animi eos ducimus debitis nulla. Sequi qui dolorum placeat, reiciendis accusantium
+                omnis quidem numquam ratione, alias ex ullam, voluptas vero voluptatem excepturi nam doloribus? Quia eius
+                obcaecati reprehenderit consequuntur laudantium dolorum, eum, iste ad in odit laboriosam aspernatur aliquid
+                eveniet consequatur minima sed vero assumenda vitae, libero ullam expedita itaque cum. Iusto, rerum totam.</p>
```

Comment:
Since you have the beginning `<p>` tag on another line, I would put the closing `</p>` tag on another line as well.